CancerGenomics.fsh

```diff
@@ -18,6 +18,7 @@ Description:    "Records an alteration in the most common DNA nucleotide sequenc
 * identifier[FillerOrderNumber].type = IDTYPE#FILL
 * identifier[PlacerOrderNumber].type = IDTYPE#PLAC
 * code = LNC#69548-6 "Genetic variant assessment"
+* method from https://fhir.loinc.org/ValueSet/LL4048-6 (extensible)
```

Comment:
I don't think this is the correct URL for LOINC lists. It should be: http://loinc.org/vs/LL4048-6. See https://www.hl7.org/fhir/loinc.html#alist for details and examples.

Reply:
changed